spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
> > Because the presence of escape sequences and line continuations—which are not exposed in ESTree—is relevant.

> They are not exposed in AST, but they are taken into account by parser which anyway generates those fields. Otherwise you could say the same about any ASI productions - line breaks are not exposed, but parser has to take them into account when generating AST.

It is a mistake to assume that ASTs are only generated by parsers, but beside the point anyway—this is about ensuring that semantically different programs don't map to identical ASTs.